platform/android/app/src/main/kotlin/coop/polypoly/polypod/homescreen/HomeScreenViewModel.kt

```diff
@@ -42,9 +45,15 @@ data class TileModel(
 data class FooterModel(
     val title: String,
     val description: String,
-    val imageId: Int,
     val buttonTitle: String,
-)
+) {
+    fun buttonOpenUri(context: Context): Uri {
+        return when (Language.determine(context)) {
+            "de" -> Uri.parse("https://polypoly.coop/de-de/becomepart")
+            else -> Uri.parse("https://polypoly.coop/en-de/becomepart")
+        }
```

Comment:
nit: this could be also part of the translations, no? so we don't need any logic part here

Reply:
Also, do we determine language here and only here? Is there not an app-wide resource that tells us so?
Since translations are such a thing, I'm kinda with @irstavr here. A single source for the language and translation strings (or URIs, or images) is possibly a good idea.